api/nakadi-event-bus-api.yaml

```diff
@@ -920,6 +920,12 @@ definitions:
               from the header into the metadata by Nakadi to avoid clients having to explicitly copy this.
             type: string
             example: 'JAh6xH4OQhCJ9PutIV_RYw'
+          partition:
+            description: |
+              This field indicates the partition to which the event should be published to. Used only for partitioning
```

Comment:
I'd replace 'Used only for ...' with 'Required when the partitioning strategy used is ...'